lib/caching-browserify.js

```diff
@@ -95,7 +95,8 @@ module.exports = CoreObject.extend({
       packageCache: {},
       fullPaths: this.fullPaths,
       basedir: this._inputStaging,
-      debug: this.enableSourcemap
+      debug: this.enableSourcemap,
+      browserField: !process.env.EMBER_CLI_FASTBOOT
```

Comment:
this mode isn't a thing anymore, we likely need to produce 2 bundles. One for fastboot and one without.

Slightly more involved, but will solve this nicely!